.github/workflows/ci.yml

```diff
@@ -910,7 +910,7 @@ jobs:
         with:
           name: debug-macos-${{ github.run_number }}
           path: debug/
-      - run: unzip debug/debug-*.zip -d debug/
+      - run: for f in debug/debug-*.zip; do unzip "$f" -d "${f%.zip}"; done
```

Comment:
@tyranron, I've discovered a problem with `unzip`ping every archive to a single directory: symbols do collide. Some symbols for `apk` and `appbundle`, `ios` and `macos` builds are named similarly, causing overwriting. Thus I suggest using `for` to retrieve the filename in a `f` variable to `unzip` archives to their respective directories.

Reply:
@SleepySquash seems to be OK.